.github/auto-comment.yml

```diff
@@ -1,16 +1,17 @@
 pullRequestOpened: >
```

Comment:
You may want to use a `|` instead of a `>` here:

> The block style indicates how newlines inside the block should behave. If you would like them to be kept as newlines, use the literal style, indicated by a pipe (|). If instead you want them to be replaced by spaces, use the folded style, indicated by a right angle bracket (>).

Reply:
!!! Where did you find this glorious piece of documentation?

Reply:
This has bit me before, so I just Googled YAML newlines. :)

Reply:
For reference, there's a whole website dedicated to the topic: https://yaml-multiline.info 😀

Reply:
Ah! I made the mistake of looking only for markdown and Github markdown. Good find, and thank you!